internal/server/ui/.gitignore

```diff
@@ -0,0 +1 @@
+static
```

Comment:
This is a bit of a trick for `go:embed`, so that it always works (even with the ui static files directory `ui/static` being empty). Consequently all generated ui files are under `ui/static` now

Reply:
It looks like `go:embed` needs the directory to not be empty. Instead of including a `.gitignore`, you could include a file `.empty` or something and generate into `ui/`. In the top level `.gitignore`, you can add an entry `ui/*` to the contents after adding `aaa/.empty` to the repo

Reply:
Yeah great point. That was my initial stab at this – although this makes `.gitignore` the `.empty` file, but also one that's handy to remove `static` from being checked in.